app/src/main/res/values/strings.xml

```diff
@@ -523,7 +523,7 @@
     <string name="installer_error_aborted">Installation failed because either because it was cancelled, or the session closed unexpectedly</string>
     <string name="installer_error_blocked_device">device</string>
     <string name="installer_error_blocked">Installation blocked by <xliff:g id="app_name" example="App Manager">%1$s</xliff:g></string>
-    <string name="installer_error_conflict">Cannot install the app because an installed app has this package name already</string>
+    <string name="installer_error_conflict">Cannot install the app because an installed one has this package name already</string>
```

Comment:
I think this needs some rework, something like _Could not install app because another app has this package name._